lib/authlogic/session/cookies.rb

```diff
@@ -35,6 +35,19 @@ def cookie_key(value = nil)
 
         # If sessions should be remembered by default or not.
         #
+        # If set to false authlogic sets session coockie that browsers normally
+        # delete when the user closes the browser and it logs out him
+        # Note: The user can configure the browser do not remove session cookies,
+        # in this case he will never be auto log out. There is this option:
+        # - 'Continue where you left off' - in Chrome
+        # - 'All windows from last session' - in Safari
+        # - 'Show my windows and tabs from last time' - in FireFox
+        #
+        # If set to true authlogic sets coockie with expiration
```

Comment:
What do you mean when say `*also*`?
As I understand Authlogic *always* creates just one cookie `user_credentials`, but when:
- remember_me is false it leaves `expires` field empty, as result we get _session cookie_
- remember_me is true it set `expires: remember_me_for`